geomstats/visualization/_plotting.py

```diff
@@ -0,0 +1,260 @@
+import abc
```

Comment:
Need docstring a the top of this file, see other geomstats' files for inspiration